pkg/retry/retry.go

```diff
@@ -38,17 +47,26 @@ func WithMaxAttempts(a int) Option {
 func Do(fn func() error, opts ...Option) error {
 	cfg := &config{
 		interval: 30 * time.Second,
+		backoff: false
```

Comment:
This is missing a comma to build properly.

You can run the following command to check if everything builds (and if lint passes):
```
make lint-go
```